pyproject.toml

```diff
@@ -17,9 +17,10 @@ readme = "README.md"
 dynamic = ["version"]
 
 dependencies = [
-    "aind-data-schema==0.15.9",
+    "aind-data-schema==0.17.3",
     "aind-metadata-service[client]",
-    "scanimage-tiff-reader==1.4.1.4"
+    "scanimage-tiff-reader==1.4.1.4",
+    "pyyaml==6.0.1"
```

Comment:
does this need to be pinned?

Reply:
Maybe? Pyyaml released breaking changes between 5.0 and 6.0 so I figured pinning was safest in case of pyyaml 7.0.